src/pickley/bstrap.py

```diff
@@ -160,38 +194,35 @@ def find_base(base):
     abort(f"Make sure '{candidates[0]}' is writeable.")
 
 
-def globally_configured_pypi_mirror(pip_conf_path="/etc/pip.conf"):
-    """
-    Best-effort parsing of /etc/pip.conf to honor globally configured mirror.
-    """
-    try:
-        import configparser
+def _groomed_mirror_url(mirror):
+    if isinstance(mirror, str):
```

Comment:
is there a missing `else`?

Reply:
No, if for any reason the argument `mirror` is not a string, the "groomed" mirror of that is `None`.